addons/gunbag/functions/fnc_toGunbagCallback.sqf

```diff
@@ -40,6 +43,6 @@ _unit removeWeapon _weapon;
 _gunbag setVariable [QGVAR(gunbagWeapon), [_weapon, _items, _magazines], true];
 
 // play sound
-if (!isNil "ACE_Backpacks") then {
+if (["ace_backpacks"] call EFUNC(common,isModLoaded)) then {
```

Comment:
Oh, if it happens on preInit instead of automatically we can't remove `isModLoaded`. Some modules don't have preInit on the first place.